### PR TITLE
[*] Command Generate - Small fixes

### PR DESCRIPTION
**Pull request by VincentMolinie**

### CODE REVIEW

File: services/dumper.js [hunk 1]
```diff
@@ -34,14 +34,15 @@ function Dumper(config) {
...
-      express: '~4.16.3',
+      chalk: '~1.1.3',
+      'cookie-parser': '1.4.4',
       debug: '~4.0.1',
       dotenv: '~6.1.0',
-      chalk: '~1.1.3',
-      sequelize: '~5.15.1',
+      express: '~4.16.3',
       [`forest-express-${orm}`]: '^5.2.0',
+      morgan: '1.9.1',
```
Comment (VincentMolinie) [inline services/dumper.js]: The only thing added here is that line. I just reorder by alphabetical order

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Command Generate - Add logs to requests on the project generated.
```
Comment (rap2hpoutre) [inline CHANGELOG.md]: Maybe

```suggestion
- Server - Add server logs on generated projects.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+
+### Changed
+- Command Generate - Update comments in routes.
```
Comment (rap2hpoutre) [inline CHANGELOG.md]: I think this is not required since it's an internal change of v3.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+
 ### Fixed
 - Command Generate - Fix smart routes for models composed of multiple words.
+- Command Generate - Rename Custom Actions to Smart Actions.
+- Command Generate - Rename Virtual to Smart.
+- Command Generate - Remove useless new lines in model and forest files.
```
Comment (rap2hpoutre) [inline CHANGELOG.md]: I think this is not required since it's an internal change of v3.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,18 @@
...
+- Technical - Drop APPLICATION_HOST in .env as it was useless.
```
Comment (rap2hpoutre) [inline CHANGELOG.md]:
```suggestion
- Technical - Drop `APPLICATION_HOST` in `.env` as it was useless.
```

File: CHANGELOG.md (hunk 3 above)
Comment (VincentMolinie) [reply CHANGELOG.md]: Yeah but that was not released with the previous PR so I still think it should be here 😉

**Pull request merged**